### PR TITLE
Add descriptions for puma dashboard graphs

### PR DESCRIPTION
**Pull request by matsimitsu**
Not sure if it's a good idea to add descriptions to all graphs, threads/workers seem self explanatory to me, but maybe I'm biased?

### CODE REVIEW

**Comment by thijsc** [comment]
> Not sure if it's a good idea to add descriptions to all graphs, threads/workers seem self explanatory to me, but maybe I'm biased?

I think a clear explanation of things that seem obvious to us still has a lot of value for people.

**Comment by matsimitsu** [comment]
> I think a clear explanation of things that seem obvious to us still has a lot of value for people.

Absolutely, it's just really difficult to come up with something that isn't a repetition of the title

File: dashboards/puma/main.yml [hunk 1]
```diff
@@ -33,6 +36,7 @@ dashboard:
           tags:
             hostname: '*'
     - title: Puma worker info
+      description: Number of workers, old workers are still running old code after a deploy and should eventually be replaced with fresh workers running new code.
```
Comment (tombruijn) [inline dashboards/puma/main.yml]:
```suggestion
      description: Number of workers per type. Old workers are still running old code after a deploy and should eventually be replaced with fresh workers running new code.
```

File: dashboards/puma/main.yml [hunk 2]
```diff
@@ -8,6 +8,7 @@ dashboard:
     and what the used/available concurrency is for those workers.
   graphs:
     - title: Threads running/max/idle
+      description: Counted over all workers, if running multiple processes.
```
Comment (tombruijn) [inline dashboards/puma/main.yml]:
```suggestion
      description: Number of threads per type counted over all workers, if running multiple processes.
```

**Pull request merged**